src/javascript/app/pages/trade/digit_trade.js

```diff
@@ -101,7 +112,8 @@ const DigitDisplay = (() => {
         if (getPropertyValue(response, ['tick', 'id']) && document.getElementById('sell_content_wrapper')) {
             ViewPopupUI.storeSubscriptionID(response.tick.id);
         }
-
+        $digit_table.find('.barspinner').remove();
+        // Hide loading component
```

Comment:
why is the comment below the line the does the action? 😅

Reply:
:joy: totally my bad.